### DISCUSSION TITLE
Unwanted vibrato/tremelo in fluidsynth

### DISCUSSION THREAD

**Discussion by equant** [GeekFunkLabs/fluidpatcher#100] [General]
I've been building a squishbox based synth.  It's great.  But I'm having an issue with fluidsynth.  I realize this isn't a fluidsynth forum, but it seems like a someone active community and attempts to figure this out elsewhere have gone unanswered.

When I play a sound, I hear a tremolo effect being added (could be vibrato I guess, but I think it's tremolo).  When I play the soundfont in polyphone, or with timidity I don't hear the tremolo effect.  This happens both on the raspberry pi squishbox version, and when I play it locally on my linux desktop.  I've tried turning off reverb and chorus and it doesn't make any difference (I don't think they're working to begin with).  I've tried setting the value of CC 1 to 0, and CC 0 to 0.  I'm not sure if I'm doing it right though.

Does anyone have any insight on this?  Is there a foolproof way to turn off everything in a bank file so you just get the bare sound?  I've tried different soundfonts.  It happens to them all.  I've created a soundfont from scratch and not added any modulators to it.  Totally stuck on how to figure this out.  Any advice is greatly appreciated.  Thank you.

**Comment by tizfra** [comment]
Hi, normally tremolo is on midi CC 92, so you can try to put that CC=0 and make sure your keyboard didn't send some CC values (for example on slider)

**Comment by albedozero** [comment]
Put this at the top of your `router_rules` section, at the bank level (i.e. outside of any patches):

```yaml
router_rules:
- clear
- {type: note, chan: 1-16*1+0}
```

The first rule clears all the default router rules, the second restores note rules for each channel so you can play.

_Vibrato_ generally means the pitch oscillates, _tremolo_ is volume oscillation. CC1 is vibrato, and is defined as a default modulator in the soundfont spec - so most soundfonts will respond to it. The MIDI spec says CC92 is tremolo, but this is not a default modulator so soundfonts will not respond to it unless they have their own custom modulators (like the modified jRhodes soundfont I include in the [squishbox soundfonts collection](https://geekfunklabs.com/download/squishbox-soundfonts/)).
